browser/src/control/Control.JSDialog.js

```diff
@@ -669,8 +675,8 @@ L.Control.JSDialog = L.Control.extend({
 		instance.canHaveFocus = !instance.isSnackbar && instance.id !== 'busypopup' && !instance.isAutoCompletePopup;
 		instance.isDocumentAreaPopup = instance.popupParent === '_POPOVER_' && instance.posx !== undefined && instance.posy !== undefined;
 		instance.isPopup = instance.isModalPopUp || instance.isDocumentAreaPopup || instance.isSnackbar;
-		instance.containerParent = instance.isDocumentAreaPopup ? document.getElementById('document-container'): document.body;
 		instance.isAutofilter = instance.isDocumentAreaPopup && this.map._docLayer.isCalc();
+		instance.containerParent = (instance.isDocumentAreaPopup && !instance.isAutofilter) ? document.getElementById('document-container'): document.body;
```

Comment:
above isAutofilter is not perfect name - it is not only autofilter but also AutoFill popup too - https://github.com/CollaboraOnline/online/blob/master/browser/src/control/AutoCompletePopup.ts

could you check if auto fill popup still works as expected?
in spreadhseet with split panes (freezed rows and columns) at some distant cell - like Z500